website/content/en/docs/cli/operator-sdk_bundle_validate.md

```diff
@@ -56,7 +56,7 @@ To build and validate an image:
 
 ```
   -h, --help                   help for validate
-  -b, --image-builder string   Tool to extract bundle image data. Only used when validating a bundle image. One of: [docker, podman] (default "docker")
+  -b, --image-builder string   Tool to pull and unpack bundle images. Only used when validating a bundle image. One of: [docker, podman, none] (default "docker")
```

Comment:
I can use podman or docker to validate the image .. however, to create a can just use docker. Should we not be a valid option to create the bundle as well?

Reply:
You can use docker, podman, or buildah to create the bundle image (see help text of `bundle create`). I think the `--image-builder` name is confusing and should be changed to `--container-tool` or something. Needs a follow-up.

Reply:
I liked `--container-tool `